README.md

```diff
@@ -55,14 +54,30 @@ developer's workstation without having to `ssh` directly into the ADB virtual ma
 
 ## Available Commands <a name="commands"></a>
 
-The following table lists the available commands for the plugin and their explanation:
+The following section lists the available commands for the plugin and their explanation:
+
+1. `vagrant service-manager env [service] [--script-readable]` 
+Displays connection information for all active services in the box in a manner that can be evaluated in a shell. If a `[service]` is specified, only the information for that service is displayed. When `--script-readable` is specified the output is in `key=value` format. 
+The supported services are: Docker; OpenShift.
+
+2. `vagrant service-manager box [command]`
+Displays box related information like release version, IP etc. 
+
+3. `vagrant service-manager box version [--script-readable]` 
+Displays the version and release information of the running VM. When `--script-readable` is specified the output is in `key=value` format.
+
+4. `vagrant service-manager box ip` 
+Displays the routable IP address of the running VM. 
+
+5. `vagrant service-manager status [service]` 
+Lists services and their running state. If a `[service]` is specified only the status of that service is displayed. If no service is provided then only supported orchestrators are reported.
+
+6. `vagrant service-manager restart [service]` 
+Restarts the given service in the box.
```

Comment:
Are you trying to show "Restarts the given service in the box." in new line?
If yes, then You need to add empty line between "6.  `vagrant service-manager restart [service]`" and "Restarts the given service in the box.". As, right now it is coming in one line check https://github.com/Preeticp/vagrant-service-manager/blob/vsm-docs-reorg/README.md#available-commands.